Wikipedia/Categories/UIView+WMFDefaultNib.m

```diff
@@ -1,17 +1,15 @@
-//
-//  UIView+WMFDefaultNib.m
-//  Wikipedia
-//
-//  Created by Brian Gerstle on 3/6/15.
-//  Copyright (c) 2015 Wikimedia Foundation. All rights reserved.
-//
 
 #import "UIView+WMFDefaultNib.h"
+#import "NSString+Extras.h"
 
 @implementation UIView (WMFDefaultNib)
 
 + (NSString*)wmf_nibName {
-    return NSStringFromClass(self);
+    /* Swift has "Namespaced" class names that prepend the module
```

Comment:
Another issue is that it won't work with any Swift classes we don't define… so we can only use the category with classes we define in Swift or include in Objective-C - starts to get harry fast.